cost/scheduled_report_markUpsDowns/README.md

```diff
@@ -0,0 +1,62 @@
+- *Cost Metric* -  See Cost Metrics above for details on selection.
+- *Compute mark up or Mark Down percentage* - mark up for the Compute category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Application Service mark up or Mark Down percentage* - mark up for the Application service category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Database mark up or Mark Down percentage* - mark up for the Database category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Network mark up or Mark Down percentage* - mark up for the Network category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Storage mark up or Mark Down percentage* - mark up for the Storage category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Reserved Instances mark up or Mark Down percentage* - mark up for the Reserved Instances category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *GENERAL MARKUP OR MARKDOWN PERCENTAGE* - general markup for the rest of the categories in number value (20 being 20% markup -20 being a 20% markdown). Any category that has 0 as a markup percentage will have this value as a Markup/down
+- *Admin mark up or Mark Down percentage* - mark up for the Admin category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Artificial Intelligence mark up or Mark Down percentage* - mark up for the Artificial Intelligence category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Application mark up or Mark Down percentage* - mark up for the Artificial Intelligence category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Marketplace mark up or Mark Down percentage* - mark up for the Marketplace category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Streaming mark up or Mark Down percentage* - mark up for the Streaming category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Support mark up or Mark Down percentage* - mark up for the support category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *IOT mark up or Mark Down percentage* - mark up for the IOT category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
+- *Other mark up or Mark Down percentage* - mark up for the Other category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markUp).
```

Comment:
You use markup and mark up and markUP and markdown and Mark Down.  Be consistent with case and spacing for both .